### DISCUSSION TITLE
Can I Simulate Trigger on External DB?

### DISCUSSION THREAD

**Discussion by tasneemimamkhan** [Budibase/budibase#1419] [Help]
Hello, 

I hope you are well!

I am having trouble adding data to multiple external mysql tables at the same time. For instance, I have a sales table, which has a column called sales leader. And I have table1 and table2. I would like to do the following:

When a new row is added to the sales table, 
if the salesleader='a', then add a new row to table1. 
If the salesleader='b', then add a new row to table2. 

I understand that creating a trigger would be the ideal solution here, however, triggers in Budibase seem to be only applied to its internal database, if I am correct. 

Is there another way I can approach and solve this problem? 

Thank you in advance!

**Comment by tasneemimamkhan** [comment]
Any help would be appreciated, thanks!

**Reply by mjashanks** [reply]
Hi There. a few suggestions, apologies if they are not all that helpful.

**Use a MySQL trigger**

I don't know much about MySQL, so don't take this as "good" advice. If you could setup a MySQL trigger to make a web request, then you could trigger a `WebHook` in Budibase. A webhook is a type of automation trigger.

A quick search, I found this: https://stackoverflow.com/questions/37215104/calling-an-url-from-a-trigger-in-mysql

**Near future feature - CRON trigger**

This feature is not developed yet, but we intend to do it soon: https://github.com/Budibase/budibase/issues/1221

With this, you could poll your MySQL database for changes, and pull in any new rows.
